source/wp-content/themes/wporg-parent-2021/sass/block-styles.scss

```diff
@@ -121,3 +121,50 @@
 		}
 	}
 }
+
+.is-style-features {
+	--wporg--style--feature--offset: 0.2em;
+	padding: 0;
+	list-style: none;
+	font-size: var(--wp--preset--font-size--heading-1);
+	font-weight: 200;
+	line-height: 1.1;
+
+	li {
+		overflow: hidden;
+		box-sizing: border-box;
+		height: 1.1em;
+		padding-top: var(--wporg--style--feature--offset);
+		padding-left: var(--wp--preset--spacing--50);
+		padding-right: var(--wp--preset--spacing--50);
+		color: #6682ff; // Hardcoded color for better contrast.
+		border-bottom: 1px solid var(--wp--preset--color--blueberry-3);
+		transition: all 0.2s ease-in-out;
+
+		a {
+			text-decoration: none;
+		}
+
+		&:hover,
+		&:focus,
+		&:focus-within {
+			padding-top: 0;
+			color: var(--wp--preset--color--blueberry-1);
+		}
+	}
+
+	// Make sure to remove padding from blocks with background colors.
```

Comment:
Tried to test this but couldn't add a background color to an `li`. Or am I misunderstanding...

Reply:
You can add a background to the list block itself in the sidebar

<img width="287" alt="Screen Shot 2022-08-04 at 11 20 28 AM" src="https://user-images.githubusercontent.com/541093/182884644-1a61eaeb-dd9f-490f-ae06-a85f8fd07601.png">